scripts/buildLogos.js

```diff
@@ -40,6 +46,12 @@ async function buildLogos() {
 
   // eslint-disable-next-line no-restricted-globals,compat/compat,promise/no-native
   await Promise.all(logoPromises);
+
+  if (shouldCommit) {
+    await $`git add cdn`;
+    await $`git commit -am "feat: generate CDN packages"`;
```

Comment:
We do stage the cdn folder in the previous command. Should we drop the `-a` from the commit operation?
```suggestion
    await $`git commit -m "feat: generate CDN packages"`;
```